GSoC-interest.md

```diff
@@ -17,26 +17,30 @@ You must complete these things by March 31, 2020 13:00 CDT. Make sure to also su
+| Georg Link | linkgeorg@gmail.com | This is an example entry | [Micro-task](https://github.com/chaoss/governance/blob/GSoCInterest-Update-2020/GSoC-Ideas.md) | [Proposal Template](https://docs.google.com/document/d/1YZez6_hgp2dBybEsMZoQ-ONB9IawK4_OPISLHe9Tjew/edit) | NO |
+| (Abhi) Abhishek Singh | abhiandthetruth@gmail.com |  Implement the Social Currency Metrics System in GrimoireLabs | [Microtask Repository](https://github.com/abhiandthetruth/Chaoss-Gsoc2020) | [Project Proposal](https://docs.google.com/document/d/1x4na_WQgmHok0ghKjfDZvtyX2-LAXFG-9V6oLjKnrkM/edit?usp=sharing) | YES |
+| Abhinav Bajpai | abhinavbajpai2012@gmail.com | Implementing GitLab Data Collection Workers & Mappers to bind GitLab API, Github API & the Augur schema | [Microtasks](https://github.com/abhinavbajpai2012/CHAOSS-GSoC-2020) | [Proposal](https://docs.google.com/document/d/1iJSvczhk7XVxgnYl0798rhiPsyxaC9LeageHjec5o8g/edit?usp=sharing) | YES |
+| Adarsh Narayan Pandey | f20180806@goa.bits-pilani.ac.in |  Implementation of GitLab data collection worker | [Microtask](https://github.com/chaoss/augur/pull/638) | [Project Proposal](https://docs.google.com/document/d/15RQ8g8y-PREJVVygLMZ2Rhk_GwRVmjEofvIx6l8-qGU/edit?usp=sharing) | YES |
+| Aditya Prajapati | aditya10699@gmail.com | GSoC Idea: Packaging and Sharing CHAOSS metrics using GrimoireLab dashboards | [Microtasks](https://github.com/Syn3rman/CHAOSS-microtasks) | [Proposal](https://docs.google.com/document/d/1H1nunFjeVjRQhtEF-VmQLaoRcb53G59eN5hIlYnTO_k/edit?usp=sharing) | YES |
+| Akshara P | aksharap.181it132@nitk.edu.in | Machine Learning for Anomaly Detection in Open Source Communities | [Microtasks repository](https://github.com/aksh555/chaoss-microtasks) | [Proposal](https://docs.google.com/document/d/16L99A3bXEsSe65ryqkC8YONOXs5ZV72_9vMApRzI06w/edit?usp=sharing) | YES |
+| Animesh Kumar | animuz111@gmail.com | Implement the Social Currency Metrics System in GrimoireLabs [#288](https://github.com/chaoss/grimoirelab/issues/288) | [Microtask Repository](https://github.com/animeshk08/gsoc-chaoss-microtasks) | [Project Proposal](https://docs.google.com/document/d/1JZbje8FfjuSybie1VUewSshTC6zJmHYPCEJ7SZNXI-8/edit?usp=sharing) | YES |
+| (bistaastha) Aastha Bist | abist119@gmail.com | Build Workflow Process for CHAOSS D&I Badging | [Microtask repository](https://github.com/bistaastha/CHAOSS-microtasks) | [Proposal](https://docs.google.com/document/d/1YPAlVUhUZG6Gc4k8l5zxZz6gWoHAG-UsLlGEOiQkvGE/edit?usp=sharing)|Manan Goel | manan.goel@research.iiit.ac.in | YES |Machine Learning for Anomaly Detection in Open Source Communities | [Repository With Completed Microtasks](https://github.com/manangoel99/augur?organization=manangoel99&organization=manangoel99) | [Proposal](https://docs.google.com/document/d/1n5byXxrmDrQkVaYFFEI9yQG3VjyaOfiFaqwGYMuTHs4/edit?usp=sharing)
```

Comment:
Hi @GeorgLink 
I just found a small inconsistency here.
The row format for Manan Goel is not aligned, I think he did not put his name in a new line which made you miss it.

I thought of putting it up.

Reply:
Thanks @vchrombie for your watchful eye. I fixed it in master. Does it look alright in the repo now?

Reply:
Ya, perfect! :smile: